collect_app/src/main/java/org/odk/collect/android/application/initialization/ApplicationInitializer.kt

```diff
@@ -108,7 +107,6 @@ class ApplicationInitializer(
                 MapView(context).onCreate(null)
             }
             Configuration.getInstance().userAgentValue = userAgentProvider.userAgent
-            MapboxUtils.initMapbox()
```

Comment:
We just don't need this any longer? Definitely nice if so!

Reply:
We don not initialize mapbox like that now. That class was also used to handle cases where a user tried to use mapbox on unsupported devices or a user didn't have a valid access token (then null was returned and we were able to handle that cases). Now as all devices are supported because I removed excluding libraries for X86 devices + without a valid downloads token you won't build the app at all. It's still possible to have a supported device + valid downloads token but wrong access token but it doesn't make sense to check such strange cases I think.